impl/src/main/java/org/jboss/cdi/tck/tests/extensions/bean/bytype/BeanByTypeTest.java

```diff
@@ -30,6 +30,7 @@
 import org.jboss.arquillian.container.test.api.Deployment;
 import org.jboss.cdi.tck.AbstractTest;
 import org.jboss.cdi.tck.shrinkwrap.WebArchiveBuilder;
+import org.jboss.shrinkwrap.api.BeanDiscoveryMode;
```

Comment:
Not needed?

Also, why is this even in the `extensions` package? It doesn't use extensions at all, does it? Is that just because `BeanManager` is in the `spi` package, or something like that?

Reply:
Don't know. I was thinking the same but didn't really know where to put it at the moment so I just left it there.